ceph.spec.in

```diff
@@ -748,19 +748,21 @@ cmake .. \
 %endif
 %endif
 
+# set PORTABLE=1 on aarch64 so rocksdb doesn't use -march=native
+# http://tracker.ceph.com/issues/18065
+%ifarch aarch64
+PORTABLE=1 make %{?_smp_mflags}
```

Comment:
rocksdb's CMakeLists.txt does not understand `PORTABLE=1`, and the `-march=native` was dropped in https://github.com/facebook/rocksdb/commit/9c0bb7f1720415c15c057787cec641ac7814a86d. 

could you try again using the latest master which includes the commit 4b2046eb686f5af9dfbff5712abf63b85c6ce1ca which fixes the cross-compiling ?